### ISSUE TITLE
Update to Python 3.11 to get lots of speed improvements

### ISSUE THREAD

**Issue by linknum23** [micro-nova/AmpliPi#415]
Maybe we could use pyenv or snap to make the switch.

**Comment by SteveMicroNova** [comment]
Bumping that this is required for #695, we currently run 3.7.3 and the tidalapi package requires 3.9 or higher to function

Going to continue devving on that branch for sure, but this is a blocker to merging in tidal support (once I get to that step)

**Comment by linknum23** [comment]
Just note that this is mostly blocked by changes to how we update.

**Comment by SteveMicroNova** [comment]
Closing as this is superseded by #995
